privacidad-margenes.md

```diff
@@ -0,0 +1,51 @@
+
+Organizarse con activistas requiere canales seguros de comunicaci�n. Eso quiere decir mantener la informaci�n secreta de todos excepto esos que requieren saberla, y hasta el momento que requieran saberlo. Mi rol en acciones directas ha sido el de apoyo, no liderazgo, lo que quiere decir que oigo sobre la ubicaci�n general del sitio unos pocos d�as antes, y la ubicaci�n exacta menos de una hora antes. Algunos grupos activistas con los que he trabajado prefieren Telegram 41, pero yo prefiero Signal42, honestamente solo me baso en la recomendaci�n de algunos de mis amigos con mayor experiencia en seguridad de la informaci�n.
+
+Como trabajadora sexual soy cuidadosa (ahora) en examinar a mis potenciales clientes para protegerme a m� misma (y, de menor forma, protegerlos a ellos). Pero la mayor�a de violencia que las trabajadoras sexuales experimentan es a manos de la polic�a y el estado, no de sus clientes.
+
+El trabajo que hago no es ilegal, pero provoca escrutinio. Tambi�n hago un esfuerzo por mantener mi �identidad� del trabajo sexual muy separada de mi identidad personal/p�blica, teniendo dispositivos independientes y cuentas y fotograf�as para cada una. He cambiado mis h�bitos de redes sociales y solo posteo sobre lugares a los que ya he ido en lugar de sitios a los que voy a ir, y por lo menos un d�a despu�s.
+
+Pero esto no es a prueba de balas. Una vez un cliente me pregunt� que c�mo estaba y, sinti�ndome particularmente orgullosa de m� misma le dije que genial porque acababa de ser publicada en The Guardian (bajo mi nombre verdadero). �l descaradamente dijo que lo sab�a, y procedi� a hablarme del tema sin darse cuenta de qu� tan inc�modo y peligroso era eso para m�. �l ahora est� en lista negra en foros de trabajadoras por ser un rarazo irrespetuoso (Rob, si est�s leyendo esto, pi�rdete).
+
+Ser trans y queer y una trabajadora sexual tambi�n me hace vulnerable a ataques personales por parte de desconocidos y �feministas� radicales en internet. Soy cuidadosa de proteger mi informaci�n personal para que el abuso transf�bico, queerf�bicos, zorraf�bico que recibo sea limitado a mis cuentas p�blicas online y no sangren a mi realidad f�sica.
+
+A las personas trans les son constantemente denegados el acceso a atenci�n m�dica en el Reino Unido. Hay muchos grupos de apoyo online que tienen variados niveles de privacidad y anonimato, y los temas m�s discutidos son c�mo navegar el sistema salud para tener acceso a hormonas u otros tratamientos de afirmaci�n de g�nero.
+
+Yo uso Signal, una aplicaci�n de mensajer�a segura y encriptada end-to-end, para discutir hormonas con personas trans que tienen dificultades para obtenerlas del sistema nacional de salud. Si soy cuidadosa, soy capaz de obtener testosterona para mis amigos que la necesitan pero no pueden obtenerla debido a que el sistema nacional de salud es muy lento y desde�oso, y los costos de servicios privados son demasiado elevados.
```

Comment:
encriptada -> cifrada
end-to-end -> de punta a punta
amigos -> amistades

Reply:
en este caso es amigos, testosterona se le da a hombres trans. Esto listo